kratos/mpi/python_scripts/distributed_gid_output_process.py

```diff
@@ -0,0 +1,77 @@
+from __future__ import print_function, absolute_import, division #makes KratosMultiphysics backward compatible with python 2.6 and 2.7
+
+import KratosMultiphysics as KM
+from KratosMultiphysics.kratos_utilities import CheckIfApplicationsAvailable
+if CheckIfApplicationsAvailable("TrilinosApplication"):
+    import KratosMultiphysics.TrilinosApplication as KratosTrilinos
```

Comment:
I would not import this here, only in the function that actually uses it

Reply:
Working with the new install structure we had some problems when importing modules inside functions. Besides this is what Pepocho suggests:

"Imports are always put at the top of the file, just after any module comments and docstrings, and before module globals and constants."

Reply:
I know, but for apps I think the situation is a bit special, I would only import them if I really use them

Besides, when having cleaned up the the things we discussed recently related to this I hope the problems with the imports will disappear